pkgs/development/web/playwright/chromium.nix

```diff
@@ -0,0 +1,30 @@
+{
+  runCommand,
+  makeWrapper,
+  makeFontsConf,
+  chromium,
+  ...
+}:
+let
+  fontconfig = makeFontsConf {
+    fontDirectories = [ ];
+  };
+in
+runCommand "playwright-browsers"
+  {
+    nativeBuildInputs = [
+      makeWrapper
+    ];
+  }
+  ''
+    mkdir -p $out/chrome-linux
+
+    # See here for the Chrome options:
+    # https://github.com/NixOS/nixpkgs/issues/136207#issuecomment-908637738
+    # We add --disable-gpu to be able to run in gpu-less environments such
+    # as headless nixos test vms.
+    makeWrapper ${chromium}/bin/chromium $out/chrome-linux/chrome \
+      --set-default SSL_CERT_FILE /etc/ssl/certs/ca-bundle.crt \
+      --set-default FONTCONFIG_FILE ${fontconfig} \
+      --add-flags "--disable-gpu"
```

Comment:
have you disabled the GPU just to pass the tests ? it could prove quite annoying for most users. I suggest to either add a `wrapperArgs` argument so that one can override the wrapper or maybe simpler, export some environment variable to disable GPU from within the nixos test. There must be one ?

Reply:
Didn't find an env var while looking for it and was unsure how to best pass browser-specific wrapperArgs through the whole machinery.

But: there's a parameter in playwright python to set command-line flags, so I adapted the test. Also, --headless seems to be good enough these days.